src/redux/books/books.js

```diff
@@ -0,0 +1,36 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+    case REMOVE_BOOK:
+      return state.filter((book) => book.id !== action.payload);
+      /*
+        use ES6 filter() method to create a new array, */
+
+      /*  which will not contain the book you want to remove from the store */
+      /*
+        (filter by the id key - i.e.:
+        return state.filter(book => book.id !== id);
+        */
+
```

Comment:
- [ ] Kindly remove commented code as it makes your codebase look messy and hard to read.